permhash/helpers.py

```diff
@@ -205,11 +241,40 @@ def create_crx_manifest_permlist(path):
     if check_type(path, CRX_MANIFEST_MIMETYPES):
         with open(path, encoding="utf-8") as manifest:
             try:
-                manifest_json = json.load(manifest)
+                manifest_text = manifest.read()
             except OSError:
-                logging.warning("Failure to load JSON from the manifest at %s", path)
+                logging.warning("This manifest is unable to be read: %s.", path)
                 return False
-            perm_list = parse_crx_manifest(manifest_json)
+            except UnicodeDecodeError:
+                try:
+                    manifest_data = open(path, encoding="ISO-8859-1")
+                    manifest_text = manifest_data.read()
+                except JSONDecodeError:
+                    logging.warning(
+                        "This manifest file is abnormal and unable to be read: %s.",
+                        path,
+                    )
+                    return False
+                except OSError:
+                    logging.warning("This manifest is unable to be read: %s.", path)
+                    return False
+            if (
+                "//" in manifest_text or "/*" in manifest_text
+            ):  # Enter to strip comments if there is suspected comments in the text.
+                manifest_json = strip_comments(manifest_text, path)
+            else:
+                try:
+                    manifest_json = json.loads(manifest_text)
+                except OSError:
```

Comment:
Same here no chance for loads to throw an OSError (https://docs.python.org/3/library/json.html)